src/hamster/lib/configuration.py

```diff
@@ -26,7 +26,7 @@
 
 import os
 from hamster.client import Storage
-from xdg.BaseDirectory import xdg_data_home
+from xdg import XDG_DATA_HOME
```

Comment:
This code fails on my system:

    Python 3.8.2 (default, Mar 05 2020, 18:58:42) [GCC] on linux
    Type "help", "copyright", "credits" or "license" for more information.
    >>> from xdg import XDG_DATA_HOME
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    ImportError: cannot import name 'XDG_DATA_HOME' from 'xdg' (/usr/lib/python3.8/site-packages/xdg/__init__.py)


OTOH, `from xdg.BaseDirectory import xdg_data_home` works for me. It seems to be yet another python incompatibility. So we'll need to do something like

    try:
        from xdg.BaseDirectory import xdg_data_home   
    except ImportError:
        try:
            from xdg import XDG_DATA_HOME as xdg_data_home
        except ImportError:
            xdg_data_home = os.environ.get('XDG_DATA_HOME')
    if xdg_data_home is None:
        xdg_data_home="{}/.local/share".format(os.environ.get("HOME"))

Not beautiful, but apparently necessary.

Reply:
I reverted that in the latest commit, thanks for pointing that out.

Reply:
Looks like this is a matter of depending on [`xdg`](https://pypi.org/project/xdg/) or [`pyxdg`](https://pypi.org/project/pyxdg/), which are two different packages for accessing xdg (and also both exporting the `xdg` top-level python package, making them incompatible I think). This PR would switch from one to the other, which @mwilck probably did not have installed.

Switching from one to the other could be done, but is probably out of scope for this PR, so good to see that @tchernobog updated the PR to just install `pyxdg` instead.

Given these two conflict, I guess that supporting both with the code @mwilck might be useful too (in case someone has `xdg` installed for another application and thus cannot install `pyxdg`), but that's also for a different PR (and mabye we should not bother until someone actually needs this). Leaving `pyxdg` as the default is probably a good idea in any case, since that seems to be hosted and/or maintained by freedesktop.org and it is also shipped by Debian (under the somewhat confusing name `python3-xdg`, which I think refers to the top-level package rather than the upstream project name)...